packages/framework/test-client-utils/src/index.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export { InsecureTokenProvider } from "@fluidframework/test-runtime-utils";
```

Comment:
This seems ok, but did you take a look at wholly moving `InsecureTokenProvider` (and maybe `InsecureUrlResolver`) to this separate package?  Or other location, maybe test-driver.

AFAICT they don't have much to do with the "runtime" so it's a bit of a weird place for them to live currently.

Reply:
`InsecureTokenProvider` gets used in a number of other locations, like `webpack-fluid-loader`, `node-host`, `iframe-host`, that don't have anything to with the "client" packages either though 😟 
Not sure if test-drivers is the best place either since that's specific for setting up driver abstractions.
We have a test-utils package as well, but most of the contents there have to do with the E2E tests.

As such, test-runtime-utils doesn't feel like too bad of a place for them to be since they are utilities to help you set up your loader to connect to the runtime

Reply:
https://github.com/microsoft/FluidFramework/issues/5917